interface/patient_file/report/custom_report.php

```diff
@@ -844,7 +844,7 @@ function next_prev(action){
                   else {
                     echo "<img src='" . $GLOBALS['webroot'] .
                       "/controller.php?document&retrieve&patient_id=&document_id=" .
-                      $document_id . "&as_file=false'><br><br>";
+                      $document_id . "&as_file=false&original_file=true&disable_exit=false&show_original=true'><br><br>";
```

Comment:
Not surprised(the controller also mangles booleans). It's not pretty, but then would keep what you have.